sdk/python/generative-ai/rag/notebooks/faiss/url_to_faiss_incremental_embeddings_with_tabular_data.ipynb

```diff
@@ -0,0 +1,963 @@
+        "    )\n",
+        "    use_automatic_compute(create_faiss_index)\n",
+        "    register_mlindex = register_mlindex_component(\n",
+        "        storage_uri=create_faiss_index.outputs.index,\n",
+        "        asset_name=asset_name\n",
+        "    )\n",
+        "    use_automatic_compute(register_mlindex)\n",
+        "    return {\n",
+        "        'mlindex_asset_uri': create_faiss_index.outputs.index,\n",
+        "        'mlindex_asset_id': register_mlindex.outputs.asset_id\n",
+        "    }"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "Now we can create the Pipeline Job by calling the `@pipeline` annotated function and providing input arguments. `asset_name` will be used when registering the MLIndex Data Asset produced by the `register_mlindex` component in the pipeline. This is how you can refer to the MLIndex within AzureML. For this job, the input data is an asset of type `URI_FOLDER` that lives in the default Datastore of the AzureML Workspace being used. The folder contains two files:\n",
+        "\n",
+        "1. MSFT_FY23Q1_10Q.docx\n",
+        "2. MSFT_FY23Q2_10Q.docx\n",
+        "\n",
+        "These files contain Microsoft's quarterly financial reports for the first and second quarter of the 2023 fiscal year. They are publicly available and are also available in the `data` folder relative to the location of this notebook file. In the below pipeline job, we only use the first one by filtering out the second one by using the glob pattern `**/*[Q1]_10Q.docx`. We do this to demonstrate the incremental embedding capability. In a later run, we will include both files and see how it affects the scenario.\n",
+        "\n",
+        "Here are screenshots of the two tables from the financial report that our model will be referencing to answer questions.\n",
+        "\n",
+        "Q1\n",
+        "![image-alt-text](data/General_and_administrative_q1.png)\n",
+        "\n",
+        "Q2\n",
+        "![image-alt-text](data/General_and_administrative_q2.png)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "gather": {
+          "logged": 1688133601533
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "asset_name = 'microsoft-earnings-fy23'\n",
+        "data_source_glob = '**/*[Q1]_10Q.docx'\n",
+        "\n",
+        "pipeline_job = urifolder_to_faiss(\n",
+        "    input_data=Input(type=AssetTypes.URI_FOLDER, path=\"azureml://datastores/workspaceblobstore/paths/msft_earnings_fy23\"),\n",
```

Comment:
This requires users to upload the test data to this location on their workspaceblobstore. Take a look at `acs_mlindex_with_langchain.ipynb` for an example of using a local path as data input, the data will automatically be uploaded for the user.